src/bun/bun.ts

```diff
@@ -1,16 +1,12 @@
-const jsonHeaders = { headers: {'Content-Type': 'application/json'} },
-    queryHeaders = { headers: {'x-powered-by': 'benchmark'} },
-    notFound = { status: 404 };
-
-function toResponse(json: Object) {
-    return new Response(JSON.stringify(json), jsonHeaders)
-}
+const queryHeaders = { headers: {'x-powered-by': 'benchmark'} },
+    notFound = { status: 404 },
+    toRes = Response.json;
 
 Bun.serve({
     fetch(req): Response | Promise<Response> {
         const pathIndex = req.url.indexOf('/', 12) + 1;
         if (pathIndex === req.url.length)
-            if (req.method === 'GET') 
+            if (req.method.charCodeAt(0) === 71) 
```

Comment:
How can we handle custom HTTP verb with this?

Reply:
@SaltyAom I changed it back
and also faster JSON parsing